### ISSUE TITLE
add auto update system

### ISSUE THREAD

**Issue by ffPjrZUGXfcxuAj** [revive-adserver/revive-adserver#335]
Add an auto update system like wordpress or an update pack with only changed file

**Comment by knofte** [comment]
Yes, this would be great. A lot of admins I'd gather (me included) is holding off with updates, since it's quite a few things to do every time there's a an upgrade.
Like now with 5.3 -> 5.3.1. If there was a webgui "update" button, it'd be so much quicker and easier. :)
